workbenchfx-core/src/main/java/com/dlsc/workbenchfx/WorkbenchSkin.java

```diff
@@ -53,10 +54,15 @@ public WorkbenchSkin(Workbench workbench) {
     initViews(workbench);
 
     getChildren().add(workbenchView);
+
+    // if there is only one module in the workbench, open it automatically
+    if (workbench.getModules().size() == 1) {
+      Platform.runLater(() -> workbench.openModule(workbench.getModules().get(0)));
+    }
   }
 
   private void initViews(Workbench model) {
-    toolbarView = new ToolbarView();
+    toolbarView = new ToolbarView(model.getModules().size() > 1);
```

Comment:
This breaks the MVP pattern and since we support adding or removing modules at runtime, this could mean that changes from 1 to 2 modules or 2 to 1 module would not be shown in the GUI. I would suggest this approach:
* don't pass anything to the view, undo the changes in the view itself
* in the `ToolbarPresenter`, add a change listener on the model's list of modules. Set the visibility of the view's bottomBox (`view.bottomBox.setVisible(...)`) from the presenter depending on the amount of modules (like you already have now). Also make sure to call `openModule(...)` if there is only one module and to call the logic setting the visibility during initialization, to cover the case when no change is being made.

This would mean that changing the amount of modules at runtime would also be applied and it would follow the MVP pattern.